backend/reg.ml

```diff
@@ -126,26 +127,24 @@ let is_reg t =
   | Reg _ -> true
   | Stack _ | Unknown -> false
 
-let reset() =
-  (* When reset() is called for the first time, the current stamp reflects
-     all hard pseudo-registers that have been allocated by Proc, so
-     remember it and use it as the base stamp for allocating
-     soft pseudo-registers *)
+let restart () =
```

Comment:
(We should maybe consider a
longer name, more in line with
the other renames. Typically
with the substring `relocatable`
in it.)